### PR TITLE
[hold for 0.68] Remove ko punches usages throughout site

### PR DESCRIPTION
**Pull request by abought**
Replaces #5224 and #5243 .
Refs https://openscience.atlassian.net/browse/OSF-5969

## Purpose
Phase out usage of knockout punches template helpers due to reported incompatibilities.

This PR modifies all pages on the entire site so that no uses of knockout punches should remain. It incorporates and updates the two PRs it is intended to replace.

## Summary of changes and testing notes
See closed PRs #5224 and #5243 for details.  This will require regression testing throughout the site to verify links, logs, display text, and button functionality.

Note to reviewers: all commits up to and including b5bc727 already passed through code review, in the two linked precursor PRs.

## Deployment notes
This PR removes a top-level mako filter- it affects the behavior of templates in a way that does not trigger an auto-rebuild. You may encounter errors when rendering mako pages, unless you clear the mako template cache:

1. Stop the server
2. Delete the contents of `/tmp/mako_modules`
3. Restart the OSF server instance

### CODE REVIEW

**Comment by abought** [comment]
Brought PR up to date and removed a few more ko punches usages that crept in.

(including some `log_templates` changes in merge commit b216134 ; sorry about that)

File: website/static/js/koHelpers.js [hunk 1]
```diff
@@ -209,6 +209,46 @@ ko.bindingHandlers.fadeVisible = {
     }
 };
 
+var fitHelper = function(value, length, replacement, trimWhere) {
```
Comment (brianjgeiger) [inline website/static/js/koHelpers.js]: If one or both of these are the code we're using from Punches, we should label it as such.

**Comment by brianjgeiger** [comment]
Looks fine. We should add in a comment or two above where we grabbed code. We've added the license already, so that's cool. Everything else is essentially the same thing over and over with one notable exception where Andy had to add a slash. But it seems fine and should be ready for some intensive QA.

**Comment by brianjgeiger** [comment]
Replaced by #5491

**Pull request closed without merging**